components/services/bat_ads/bat_ads_impl.cc

```diff
@@ -91,8 +91,8 @@ void BatAdsImpl::OnTimer(const uint32_t timer_id) {
   ads_->OnTimer(timer_id);
 }
 
-void BatAdsImpl::OnUnIdle() {
-  ads_->OnUnIdle();
+void BatAdsImpl::OnUnIdle(const uint64_t idle_time, const bool was_locked) {
```

Comment:
Totally a nitpick, but maybe `OnActive` would be a better name for this?

Reply:
We used the same naming convention from Chromium

Reply:
OK, was thinking that must be it - works for me then!